### PR TITLE
Added SMI import and unittest

### PR DESCRIPTION
**Pull request by Kohulan**

### CODE REVIEW

File: CIDER/cider.py [hunk 1]
```diff
@@ -230,6 +230,55 @@ def import_as_data_dict(self, path_to_data: str) -> Dict:
...
+            all_dicts (dict): Dictionary with subdictionaries for every dataset, including the key 'self.import_keyname'.
+
+        Raises:
+            FileNotFoundError: if the data path is invalid.
+            KeyError: if no SMI Files are in the given directory.
+        """
+        all_dicts = {}
+        data_dir = os.path.abspath(str(path_to_data))
+        for dict_name in os.listdir(data_dir):
+            if dict_name[-3:] == "smi" or dict_name[-3:] == "SMI":
```
Comment (JonasSchaub) [inline CIDER/cider.py:250]: Sometimes, SMILES files are also simply stored as .txt. I would therefore include that here. What do the others think?

Comment (Kohulan) [reply CIDER/cider.py:250]: I would agree with you. Most people also store SMILES without any extension as well.

**Comment by Kohulan** [comment]
All tests are passing no issues reported!

File: CIDER/cider.py (hunk 1 above)
Comment (JonasSchaub) [reply CIDER/cider.py:250]: How can you store a file without *any* extension? It would be a text file then, right?

**Comment by hannbus** [comment]
The IDs are not saved so far and the get_database_id() cannot find the IDs because they are stored in a different way in comparison to the SDFs. (The function does not fail, but informs that no IDs could be found.)
It would be possible to save the IDs while importing. But there is no control to check, if actuel IDs are saved or another no-ID value from the file. Therefore I would make the ID import an option to use, when it is confirmed that there are IDs in the files.
The .txt extention does also work with the function if the if-statement is changed.

**Comment by JonasSchaub** [comment]
Ok, the behaviour of the get_database_id() function is exactly right, then.

And yes, there is no (easy) way to check whether something is an id or not. So your idea with the option is a good one!

**Comment by hannbus** [comment]
I added the txt extention. The ID from the smi/txt files can now also be imported, as long as the ID and SMILES strings are the first and second column (the order does not matter and is does not matter if the order is different between different files). The SMILES alone can be imported no matter in which column there are.

**Comment by JonasSchaub** [comment]
Is there anything keeping us from merging here? I got notified because Hannah added two commits after I approved the changes and the pull request was not accepted in the mean time.

**Pull request merged**